src/Components/Activities/ActivitiesList.js

```diff
@@ -1,32 +1,15 @@
 import React from 'react';
+import Title from '../Title/Title';
+import PaginationActivities from './ActivitiesPagination/ActivitiesPagination';
 import '../CardListStyles.css';
-
 const ActivitiesList = () => {
```

Comment:
if you make the ActivitiesPagination component generic you should get the activities from the service here and create a function to return the cards:
```
const showItemsListComponent = (items) => (<ActivitiesCards activities={items} />);
```